privx_api/connection_manager.py

```diff
@@ -60,7 +60,7 @@ def search_connections(
         response_status, data = self._http_post(
             UrlEnum.CONNECTION_MANAGER.SEARCH,
             query_params=search_params,
-            body=connection_params,
+            body=connection_params or {},
         )
         return PrivXAPIResponse(response_status, HTTPStatus.OK, data)
 
```

Comment:
There is nothing wrong with or operator but I would recommend this:

```
# Put this in some utils module
def not_none(obj, devault_value):
    return obj if obj is not None else default_value

# usage. Added benefit is this could be reused for other not-none check
search_params = not_none(search_params, dict())
# then keep the function call simple without inline conditional assignment
```